src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,5 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+export const UserInfo = ({ user: { name, email, username } }) => (
+  <a className="UserInfo" href={`mailto:${email || 'email unidentified'}`}>
```

Comment:
In the href attribute, it is suggested to use a more descriptive message in case the email is not provided. 'email unidentified' might not be very helpful for the user.